LabExT/View/ToDoTable.py

```diff
@@ -115,13 +117,20 @@ def regenerate(self):
 
         Final step is to sort all displayed items in the treeview and adapt their displayed index.
         """
+        self._meas_control_settings.update()
 
         # get list of hashes from currently displayed items
         leftover_hashes = list(self._tree.get_children(item=""))
 
+        n = len(self._original_todo_list)
+        max_displayed_half = self._meas_control_settings.max_displayed_todo // 2
         # compare SOLL vs IST list of what we need to display
         for tidx, todo in enumerate(self._original_todo_list):
 
+            if self._meas_control_settings.displayed_todo_limited:
+                if max_displayed_half - 1 < tidx < n - max_displayed_half:
+                    continue
```

Comment:
Too simple implementation.

Non-plus-ultra would be a scrollable selection of which ToDo's to display.

In the minimum though, I'd display a row of just `...` to show the user that there are more (not-displayed) Todos.

Reply:
Thank you for this suggestion. I added one row of `...` to replace all Todos which are not displayed.